<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Avatar</class>
 <widget class="QWidget" name="Avatar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>252</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>214</width>
    <height>252</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Avatar_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/Image/Images/Avatar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SetAvatar_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>выбрать изображение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeletepushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
